cereal/log.capnp

```diff
@@ -2479,6 +2479,11 @@ struct Microphone {
   filteredSoundPressureWeightedDb @2 :Float32;
 }
 
+struct AudioData {
```

Comment:
let's include a monotonic count too so we can piece together audio from routes with missing segments?